types/canvas.d.ts

```diff
@@ -0,0 +1,100 @@
+import * as THREE from 'three'
```

Comment:
These `*.d.ts` files are generated by the `npm run typegen` script, and should be available for consumption by end-users. TypeScript is also pretty smart, and will generate inferred types, without any explicit types in the source code.